content/applications/inventory_and_mrp/quality/quality_check_types/picture_check.rst

```diff
@@ -124,30 +125,44 @@ specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a
 specified, a *Take a Picture* quality check is created for that specific work order, rather than the
 manufacturing order as a whole.
 
-*Take a Picture* quality checks created for work orders must be completed from the tablet view. To
-do so, begin by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing
-Orders`. Select a manufacturing order that includes a work order for which a quality check is
-required. Open the tablet view for that work order by selecting the :guilabel:`Work Orders` tab, and
-then clicking the :guilabel:`📱 (tablet)` button on the order's line.
+*Take a Picture* quality checks configured for work orders must be completed from the *Shop Floor*
+module. To do so, begin by navigating to :menuselection:`Manufacturing --> Operations -->
+Manufacturing Orders`. Select an |MO| that includes a work order for which a *Take a Picture*
```

Comment:
since you defined "take a picture" in the previous sentence, this instance doesn't need the capitalization and italics. However, I understand if you decide to keep this formatting, since I'm not quite sure how else to format it. Maybe rewrite the sentence to something like

"Select an |MO| with a quality check that requires taking a picture."?